website/content/en/docs/helm/reference/advanced_features.md

```diff
@@ -96,7 +96,16 @@ $ operator-sdk exec-entrypoint helm --max-workers 10
 
 ## Use `helm upgrade --force` for deployment
 
-By adding the annotation `helm.operator-sdk/upgrade-force: "True"` to the deployed CR, the operator uses the `force` flag of helm to replace the rendered resources. For more info see the [Helm Upgrade documentation](https://helm.sh/docs/helm/helm_upgrade/).
+By adding the annotation `helm.operator-sdk/upgrade-force: "True"` to the deployed CR, the operator uses the `force` flag of helm to replace the rendered resources. For more info see the [Helm Upgrade documentation](https://helm.sh/docs/helm/helm_upgrade/) and the [helm issue](https://github.com/helm/helm/issues/1958) where the `--force` flag was originated.
```

Comment:
I was testing this out trying to change a service from ClusterIP to NodePort, and the upgrade is failing even with the force option enabled. Digging into this further, I found that the behavior changed in Helm 3. See https://github.com/helm/helm/issues/7082#issuecomment-559558318.

So I don't think we should link to the original PR that was for Helm 2 since that is no longer valid. Unfortunately I can't find any official documentation about how --force works in Helm 3. In the absence of that, I suppose linking to the comment I linked is probably fine.